### ISSUE TITLE
Styles: Visual regression tests for the text input

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#1165]
Write visual regression tests for the text [input component](https://next.design-system.post.ch/?path=/story/components-text-input--default).

Create a storybook page similar to [the button test page](https://next.design-system.post.ch/?path=/story/hidden-button-snapshot-test--page) that shows all variants on one page. In the snapshot test, navigate to this page and take one snapshot of all variants at once.

```[tasklist]
### Properties to test
- [x] Label: Short and long text
- [x] Floating label
- [x] Hidden label
- [x] Type
- [x] Size
- [x] Helper text: Short and long text
- [x] Disabled
- [x] Validation
- [x] Viewports: 320, 1024
```

**Comment by imagoiq** [comment]
@b1aserlu Percy show blank screen, can you have a look, please?

https://percy.io/3f971362/design-system/builds/30588887/changed/1690054624?browser=firefox&browser_ids=47%2C48%2C49&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1024&widths=320%2C400%2C600%2C1024%2C1440


e.g.
![image](https://github.com/swisspost/design-system/assets/12294151/92e58245-c47b-4ec2-b422-5951194d139d)

**Comment by b1aserlu** [comment]
I saw that I forgot to update the list of types and I now wait for the textarea to be visible. I hope that this will fix the problem.
The fix is in this PR #2095
